mobile-ui/src/main/java/org/sagebionetworks/research/mobile_ui/show_step/view/ShowCompletionStepFragment.java

```diff
@@ -67,4 +91,35 @@ protected int getLayoutId() {
     protected UIStepViewBinding<CompletionStepView> instantiateAndBindBinding(final View view) {
         return new UIStepViewBinding<>(view);
     }
+
+    @Override
+    protected void update(@NonNull CompletionStepView stepView) {
```

Comment:
I think that this code might belong in an mPower specific completion step. Do we want replacing %@'s in the title and text to be a part of the default functionality in SageResearch?

Reply:
No, we don't. I don't think CompletionStep belongs in SageResearch. A generic one could go in BridgeAndroidSDK's SageResearch module and an mPower one in motor control module.

@syoung-smallwisdom

Reply:
I would agree that this is pretty specific to mPower. For both mPower and CRF, we kept the completion steps in the app framework.